examples/go-tc-counter/main.go

```diff
@@ -71,196 +42,89 @@ func main() {
-			bytecodeLocation = configData.Config.BytecodeLocation
-			bytecodeSrc = srcLocation
-		} 
-	}
-
-	if bytecodeSrc == srcUuid {
-		log.Printf("Using Input: Interface=%s Source=%s",
-			iface, id)
-	} else {
-		log.Printf("Using Input: Interface=%s Priority=%d Source=%s",
-			iface, priority, bytecodeLocation)
-	}
-
-	// If the bytecode src is a UUID, skip the loading and unloading of the bytecode.
-	if bytecodeSrc != srcUuid {
-		ctx := context.Background()
-
-		creds, err := loadTLSCredentials(configData.Tls)
-		if err != nil {
-			log.Printf("Failed to generate credentials for new client: %v", err)
-			return
-		}
 
-		// Set up a connection to the server.
-		conn, err := grpc.DialContext(ctx, "localhost:50051", grpc.WithTransportCredentials(creds))
-		if err != nil {
-			log.Printf("did not connect: %v", err)
-			return
-		}
-		c := gobpfd.NewLoaderClient(conn)
-
-		loadRequest := &gobpfd.LoadRequest{
-			Location: bytecodeLocation,
-			SectionName: "stats",
-			ProgramType: gobpfd.ProgramType_TC,
-			AttachType: &gobpfd.LoadRequest_NetworkMultiAttach{
-				NetworkMultiAttach: &gobpfd.NetworkMultiAttach{
-					Priority: int32(priority),
-					Iface:    iface,
-					Direction:   direction,
+			// Set up a connection to the server.
+			conn, err := grpc.DialContext(ctx, "localhost:50051", grpc.WithTransportCredentials(creds))
+			if err != nil {
+				log.Printf("did not connect: %v", err)
+				return
+			}
+			c := gobpfd.NewLoaderClient(conn)
+
+			loadRequest := &gobpfd.LoadRequest{
+				Location: paramData.BytecodeLocation,
+				SectionName: "stats",
+				ProgramType: gobpfd.ProgramType_TC,
+				AttachType: &gobpfd.LoadRequest_NetworkMultiAttach{
+					NetworkMultiAttach: &gobpfd.NetworkMultiAttach{
+						Priority:  int32(paramData.Priority),
+						Iface:     paramData.Iface,
+						Direction: paramData.Direction,
+					},
 				},
-			},
-		}
-
-		// 1. Load Program using bpfd
-		var res *gobpfd.LoadResponse
-		res, err = c.Load(ctx, loadRequest)
-		if err != nil {
-			conn.Close()
-			log.Print(err)
-			return
-		}
-		id = res.GetId()
-		log.Printf("Program registered with %s id\n", id)
+			}
 
-		// 2. Set up defer to unload program when this is closed
-		defer func(id string) {
-			log.Printf("Unloading Program: %s\n", id)
-			_, err = c.Unload(ctx, &gobpfd.UnloadRequest{Id: id})
+			// 1. Load Program using bpfd
+			var res *gobpfd.LoadResponse
+			res, err = c.Load(ctx, loadRequest)
 			if err != nil {
 				conn.Close()
 				log.Print(err)
 				return
 			}
-			conn.Close()
-		}(id)
+			paramData.Uuid = res.GetId()
+			log.Printf("Program registered with %s id\n", paramData.Uuid)
+
+			// 2. Set up defer to unload program when this is closed
+			defer func(id string) {
+				log.Printf("Unloading Program: %s\n", id)
+				_, err = c.Unload(ctx, &gobpfd.UnloadRequest{Id: id})
+				if err != nil {
+					conn.Close()
+					log.Print(err)
+					return
+				}
+				conn.Close()
+			}(paramData.Uuid)
+		}
+
+		// 3. Get access to our map
```

Comment:
This is a real nit, and I copied the comment here from the xdp example, so it's the same there, but we're really not getting access to our map here.  I would change it to something like "Set the path to our map".